src/scss/_mixins.scss

```diff
@@ -1,65 +1,81 @@
-/// Output All oMessage Features
-@mixin oMessage($class: $o-message-class, $types: 'all', $status: 'all') {
-	@each $type in $types {
-		@if $type == 'all' {
-			@include oMessageAlerts($class, $type, $status);
-			@include oMessageNotices($class, $type, $status);
-			@include oMessageActions($class, $type, $status);
+/// Inner Message base
+///
+/// Outputs base styles for an inner variant of a message
+/// @access private
+@mixin _oMessageInner() {
+	.o-message.o-message--inner {
+		.o-message__container {
+			padding: oTypographySpacingSize(2) oTypographySpacingSize(5) oTypographySpacingSize(2) oTypographySpacingSize(4);
+
+			&:before {
+				top: unset;
+			}
+
+			.o-message__close {
+				right: oTypographySpacingSize(4);
+			}
+		}
 
-		} @else if index($o-message-alert-types, $type) {
-			@include oMessageAlerts($class, $type, $status);
+		.o-message__content {
+			@include oTypographyMargin($top: 3, $bottom: 0);
+			max-width: oTypographyMaxLineWidth($o-message-typography-scale);
 
-		} @else if index($o-message-notice-types, $type) {
-			@include oMessageNotices($class, $type, $status);
-		} @else if index($o-message-action-types, $type) {
-			@include oMessageActions($class, $type, $status);
+			.o-message__content-main {
+				@include oTypographyMargin($top: 0, $bottom: 3);
+			}
+		}
 
-		} @else {
-			@include _oMessageError("o-message types don't support #{$type}.
-			The available types are:
-			- action
-			- alert
-			- alert-bleed
-			- alert-inner
-			- notice
-			- notice-bleed
-			- notice-inner");
+		.o-message__actions {
+			@include oTypographyPadding($bottom: 3);
 		}
 	}
 }
 
-
-/// Message base
+/// Type styling
 ///
-/// Outputs base styles for a Message
-/// @param {String} $class  - base class for message
+/// Outputs base styles for an type of message
+/// @access private
+@mixin _oMessageType($type) {
+	.o-message--#{$type} {
+		@if $type == 'alert' {
+			.o-message__content {
+				padding-left: $_o-message-alert-icon-size * 1px;
+			}
 
-@mixin oMessageBase($class) {
-	.#{$class} {
-		.#{$class}__container {
-			@include oMessageContainer;
-		}
-	}
+		} @elseif $type == 'notice' {
+			.o-message__content {
+				padding-left: $_o-message-alert-icon-size / 2 * 1px;
+			}
 
-	.#{$class}--closed {
-		display: none;
-	}
-};
+		} @elseif $type == 'action' {
+			.o-message__content {
```

Comment:
Looks like the action message is different on small viewports. Previously the action would become left aligned.

(this PR / current)
<img width="408" alt="screen shot 2018-11-26 at 11 33 35" src="https://user-images.githubusercontent.com/10405691/49011851-774d0400-f16f-11e8-8c11-d5ba5517fcda.png">

Reply:
Whoops was supported to start a review.